Sources/Sentry/Public/SentryOptions.h

```diff
@@ -2,7 +2,15 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
-@class SentryDsn, SentrySdkInfo;
+@class SentryDsn, SentrySdkInfo, SentryTransactionSamplingContext;
+
+/** Function pointer for a sampler callback.
+ *
+ * @param samplingContext context of the sampling.
+ *
+ * @return A sample rate between 0.0 and 1.0.
+ */
+typedef double (*SentryTraceSampler)(SentryTransactionSamplingContext *samplingContext);
```

Comment:
`m`: We have all typedef in `SentryDefines.h`

Reply:
`h`: I think this should be `nullable`, right @marandaneto? See https://github.com/getsentry/sentry-java/blob/4620134a84de5e8a10a2811392f31941a10059cc/sentry/src/main/java/io/sentry/SentryOptions.java#L1300-L1301

Reply:
`h`: I think the name should be
```suggestion
typedef double (*SentryTracesSamplerCallback)(SentryTransactionSamplingContext *samplingContext);
```